integration_tests/src/main/python/asserts.py

```diff
@@ -380,6 +380,15 @@ def assert_gpu_and_cpu_are_equal_collect(func, conf={}, is_cpu_first=True):
     """
     _assert_gpu_and_cpu_are_equal(func, 'COLLECT', conf=conf, is_cpu_first=is_cpu_first)
 
+def assert_gpu_and_cpu_are_equal_collect_debug(df_fun, result_fun, conf={}, debug=False, is_cpu_first=True):
```

Comment:
The name of this function is `_debug`, so why would I ever want to pass `debug=False`, and why would that be a default?